src/compiler/diagnosticMessages.json

```diff
@@ -5702,6 +5702,42 @@
         "category": "Message",
         "code": 6283
     },
+    "File Layout": {
+        "category": "Message",
+        "code": 6284
+    },
+    "Environment Settings": {
+        "category": "Message",
+        "code": 6285
+    },
+    "See also https://aka.ms/tsconfig_modules": {

```

Comment:
I feel bad but is there a way to not have an underscore in the link? 😅

Reply:
I can make another aka. What do we want?